selfdrive/car/hyundai/values.py

```diff
@@ -1414,19 +1414,22 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
       b'\xf1\x8758520-K4010\xf1\x00OS IEB \x04 101 \x11\x13 58520-K4010',
       b'\xf1\x8758520-K4010\xf1\x00OS IEB \x03 101 \x11\x13 58520-K4010',
       b'\xf1\x00OS IEB \r 102"\x05\x16 58520-K4010',
+      b'\xf1\x00OS IEB \x02 102"\x05\x16 58520-K4010',
     ],
     (Ecu.fwdCamera, 0x7C4, None): [
       b'\xf1\x00OSP LKA  AT CND LHD 1.00 1.02 99211-J9110 802',
       b'\xf1\x00OSP LKA  AT EUR RHD 1.00 1.02 99211-J9110 802',
       b'\xf1\x00OSP LKA  AT AUS RHD 1.00 1.04 99211-J9200 904',
       b'\xf1\x00OSP LKA  AT EUR LHD 1.00 1.04 99211-J9200 904',
       b'\xf1\x00OSP LKA  AT EUR RHD 1.00 1.04 99211-J9200 904',
+      b'\xf1\x00OSP LKA  AT USA LHD 1.00 1.04 99211-J9200 904',
```

Comment:
@sunnyhaibin another project if you have some extra time. We're not running our specialized fuzzy FW function on this car because there's no plain text date code in the fwdCamera FW, but it's possible it's somewhere else.

Reply:
Actually there's three bytes in the ABS firmware that could be something... Two unique values:

` \x11\x13` and `"\x05\x16` which in hex is: `201113` and `220516`, both conveniently valid dates...

Reply:
Ha! On our Sonata ABS translates to `190801` and fwdCamera has in plain text: `191016`

Reply:
Interesting, I'll investigate in the near future :)